sdk/tables/azure-data-tables/README.md

```diff
@@ -51,9 +316,17 @@ This project has adopted the [Microsoft Open Source Code of Conduct][coc]. For m
 [coc_contact]: mailto:opencode@microsoft.com
 [coc_faq]: https://opensource.microsoft.com/codeofconduct/faq/
 [coc]: https://opensource.microsoft.com/codeofconduct/
+[cosmosdb_create_cli]: https://docs.microsoft.com/en-us/azure/cosmos-db/scripts/cli/table/create
+[cosmosdb_create_portal]: https://docs.microsoft.com/en-us/azure/cosmos-db/create-table-java#create-a-database-account
+[jdk]: https://docs.microsoft.com/java/azure/jdk/
+[log_level]: https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/core/azure-core/src/main/java/com/azure/core/util/logging/LogLevel.java
 [package]: https://search.maven.org/artifact/com.azure/azure-data-tables
-[samples_readme]: src/samples/README.md
+[product_documentation]: https://docs.microsoft.com/en-us/azure/cosmos-db/table-storage-overview
```

Comment:
I'd remove the `en-us` so we can let user's browser locale decide where to go.

Reply:
Fixed.